code/modules/mob/dead/new_player/new_player.dm

```diff
@@ -371,6 +373,21 @@
 					if(template.limit <= count)
 						alert(src, "The ship limit of [template.limit] has been reached this round.")
 						return
+		//Password creation
+		if (template.disable_passwords == 0)
+			var/password_choice = tgui_alert(src, "Enable password protection for [password_cost] voidcoins", "Password Protection", list("Yes", "No"))
+			//Withdraw coins
+			if(SSdbcore.IsConnected() && usr.client.get_metabalance() < password_cost)
+				alert(src, "You have insufficient metabalance to cover this purchase! (Price: [password_cost])")
+				return
```

Comment:
return the proc again because we want to just go back to the main menu, instead of making them click the join game button again
```suggestion
				return LateChoices()
```